change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW

```diff
@@ -709,14 +712,25 @@ PROCEDURE GETNAMELOCS
 ** information for allowed name types and levels. Only
 ** called if old address matching is enabled.
 *]
+ FOR EACH NAME WITH (NAME:TYPE=0)
+  DO
+   OLDSTREET=NAME:STREET
+   OLDEXTRA=NAME:EXTRAADDRESS
+   OLDCITY=NAME:CITY
+   OLDSTATE=NAME:STATE
+   OLDZIP=NAME:ZIPCODE
```

Comment:
Since everywhere (that I can see) is comparing only the first 5 characters of the 'OLDZIP', why not just store the first 5 characters here and eliminate the need for the system to recalculate the "SEGMENT(OLDZIP,1,5)" value multiple times below (for each found name record)?